gff/models/two_backbones.py

```diff
@@ -430,4 +444,7 @@ def forward(self, ex):
             out = model(ex)
             assert out.shape == (8, 3, 224, 224)
             print("🗸", end="")
+
+    print("Peter checking weight importance for LR")
+    lr_model.print_weights(model, feature_names=["S1 (0)", "S1 (1)", "Local DEM"])
```

Comment:
This shouldn't be here. At the very least, it should be in an if-statement.

But, I don't want a print statement with your name in it in the final code base. :laughing:

Reply:
bouuuuh :p

 fair enough ^^